pkg/services/ngalert/api/api_alertmanager.go

```diff
@@ -43,7 +43,7 @@ func (e UnknownReceiverError) Error() string {
 }
 
 func (srv AlertmanagerSrv) RouteGetAMStatus(c *contextmodel.ReqContext) response.Response {
```

Comment:
Changes in this file:
- Passing the context from the request to `AlertmanagerFor()`
- Receiving a context in `AlertmanagerFor()` and passing it down to the multi-org Alertmanager